Mathlib/Data/Real/EReal.lean

```diff
@@ -1093,8 +1175,8 @@ lemma add_le_of_forall_lt {a b c : EReal} (h : ∀ a' < a, ∀ b' < b, a' + b' 
 lemma le_add_of_forall_gt {a b c : EReal} (h₁ : a ≠ ⊥ ∨ b ≠ ⊤) (h₂ : a ≠ ⊤ ∨ b ≠ ⊥)
     (h : ∀ a' > a, ∀ b' > b, c ≤ a' + b') : c ≤ a + b := by
   rw [← neg_le_neg_iff, neg_add h₁ h₂]
-  exact add_le_of_forall_lt fun a' ha' b' hb' ↦ le_neg_of_le_neg
-    <| (h (-a') (lt_neg_of_lt_neg ha') (-b') (lt_neg_of_lt_neg hb')).trans_eq
+  exact add_le_of_forall_lt fun a' ha' b' hb' ↦ EReal.le_neg_of_le_neg
```

Comment:
Although I appreciate the synthesis, I think that expanding this proof a bit might help. In particular, constructing the huge function term in the `exact` is a bit cheating, no?